pizzeria/styles/media.css

```diff
@@ -0,0 +1,42 @@
+@media screen and (max-width: 389px) {
```

Comment:
too low
you have ugly view on bigger screens 460px for example

<img width="482" alt="image" src="https://github.com/MishaMazniak/Robot_Dreams/assets/13466381/c09b4216-07df-4b23-908d-e7013efaed99">